src/getHumanAge.test.js

```diff
@@ -5,4 +5,44 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
-});
+
+  test('catAge and dogAge should return 0 if humanAge is 0 for both', () => {
```

Comment:
```suggestion
  test('catAge and dogAge should return [0, 0] if humanAge is 0 for both', () => {
```

because now it seems like the function returns a number, not an array